include/seqan/index/index_fm.h

```diff
@@ -205,6 +205,16 @@ class Index<TText, FMIndex<TSpec, TConfig> >
     typename Fibre<Index, FibreLF>::Type            lf;
     typename Fibre<Index, FibreSA>::Type            sa;
 
+    /*!
+     * @fn FMIndex::Index
+     * @brief Constructor
+     *
+     * @signature Index::Index();
+     * @signature Index::Index(text);
+     *
+     * @param[in] text The text to be indexed.
+     */
```

Comment:
Or we could refer to ModifiedStrings.

I think @h-2 uses the reverse function in Lambda. Maybe he can explain 
why he doesn't use a ModifiedString instead. Performance reasons?

On 04.07.2016 22:34, smehringer wrote:
> In include/seqan/index/index_fm.h
> <https://github.com/seqan/seqan/pull/1777#discussion_r69489990>:
>
>> @@ -205,6 +205,16 @@ class Index<TText, FMIndex<TSpec, TConfig> >
>>      typename Fibre<Index, FibreLF>::Type            lf;
>>      typename Fibre<Index, FibreSA>::Type            sa;
>>
>> +    /*!
>> +     * @fn FMIndex::Index
>> +     * @brief Constructor
>> +     *
>> +     * @signature Index::Index();
>> +     * @signature Index::Index(text);
>> +     *
>> +     * @param[in] text The text to be indexed.
>> +     */
>
> so shall I change the documentation?
> |Index::Index(revText)|
> |revText the reversed text to be indexed|
>
> maybe with a link to the |reverse| function for strings?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/seqan/seqan/pull/1777/files/100e47a0d34eed6e632e33c3201c190efa1a0882#r69489990>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG_SM2yOiNcT4M12DkXmSIdw7qK9lcj5ks5qSW5GgaJpZM4JCMd1>.
>